src/storage-client/src/controller.rs

```diff
@@ -1010,6 +1022,26 @@ where
                             self.truncate_managed_collection(id).await;
                             self.initialize_shard_mapping().await;
                         }
+                        IntrospectionType::StorageHostMetrics => {
+                            // The metrics scraper assumes that the collection
+                            // sums up to empty when starting up. When needed,
+                            // we can change the scraper to reconcile its
+                            // internal state with the current state of the
+                            // output collection.
+                            self.truncate_managed_collection(id).await;
```

Comment:
This is quite subtle, and likely requires even more comments:

1. it takes the collection to empty, at roughly `now()`, but doesn't delete the old data (that we definitely want to be able to see!
2. It uses the `CollectionManager`, once which requires that this collection is register to `PersistWriteHandles`, which it is, above
3. The `CollectionManager` manages writes and bumping the upper, the metric task simply produces the values that are send to the manager.